lib/dekiru/data_migration_operator.rb

```diff
@@ -13,6 +13,7 @@ def self.execute(title, options = {}, &block)
     def initialize(title, options = {})
       @title = title
       @options = options
+      @logger = @options[:logger] || Logger.new(Rails.root.join("log/data_migration_#{title.gsub(/[[:space:]]/, '')}.log"))
```

Comment:
tilteそのままファイル名に使うとescape周りが面倒そうだから（↑だと怖そう）、titleそのまま使う必要性が低いのであれば`data_migration_実行日時.log`とかのほうが安心で良さそうだなと思いました。

Reply:
あとこれだとログを無効化するのが面倒なので、↓みたいなほうが良さそう。（`logger: nil`で済むように）
他のオプション初期化コードとも揃うし。

```suggestion
      @logger = @options.fetch(:logger) { Logger.new(Rails.root.join("log/data_migration_#{title.gsub(/[[:space:]]/, '')}.log")) }
```